typescript/ws-protocol-examples/README.md

```diff
@@ -1,22 +1,37 @@
-# Foxglove Studio WebSocket protocol examples
+# Foxglove WebSocket examples
 
-This package provides example server and client scripts using the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol).
+The [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol) supports any data encoding, as long as it is supported by both client and server. This package provides example server and client implementation of the protocol.
 
-## Usage
+## Example servers
+
+Run the `sysmon` example server, which uses JSON-encoded data to transmit system monitoring information like memory and CPU usage:
 
 ```
-$ npx @foxglove/ws-protocol-examples --help
-$ npx @foxglove/ws-protocol-examples image-server
 $ npx @foxglove/ws-protocol-examples sysmon
-$ npx @foxglove/ws-protocol-examples simple-client [host]
 ```
 
-To see data from the example servers, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
+Run the `image-server` example server, which uses JSON-encoded data to transmit images:
```

Comment:
Not really a big deal, but I think if you try to run both servers at once, the second one will fail because the port is already in use. Wonder if there is anything we can do with the wording/structure here to subtly encourage people to run one at a time.

Reply:
```
You must close out each server (`control` + `c`) before starting up the other.

To see data from either server, open [Foxglove Studio](https://studio.foxglove.dev?ds=foxglove-websocket&ds.url=ws://localhost:8765/) with a Foxglove WebSocket connection to `ws://localhost:8765/`:
```